test/uk/gov/hmrc/helptosavestub/controllers/SquidControllerSpec.scala

```diff
@@ -146,20 +146,26 @@ class SquidControllerSpec extends UnitSpec with WithFakeApplication {
     Json.toJson(goodAccount copy (contactDetails = cd))
   }
 
-  private val fakeRequest = FakeRequest().withJsonBody(Json.toJson(goodAccount)).withHeaders((CONTENT_TYPE, "application/json"))
+  private val fakeRequest = FakeRequest().withJsonBody(Json.toJson(goodAccount)).withHeaders((CONTENT_TYPE, "application/json"), "Authorization1" -> "someValue")
```

Comment:
These tests will fail if we ever change the authorisation header key in the `application.conf` - it's safer to read this from the config here if that's what's happening in the controller